src/common/System.cpp

```diff
@@ -391,9 +391,17 @@ void* operator new(size_t n)
 		Sys::Error("Out of memory");
 	return p;
 }
+
 void operator delete(void* p) NOEXCEPT
 {
 	if (!Sys::processTerminating) {
 		free(p);
 	}
 }
+
```

Comment:
makes me thinking about 3 (not related to C++14) things:

1. there is no new[] / delete[] operators
2. builds don't pass when using USE_ADDRESS_SANITIZER

Not really the point of the PR, though.

Reply:
This makes me think that for ASAN we could simply encapsulate the whole block like [this](https://clang.llvm.org/docs/AddressSanitizer.html#conditional-compilation-with-has-feature-address-sanitizer):
```C++
#if !defined(__has_feature) || !__has_feature(address_sanitizer)
// code that builds only when AddressSanitizer is disabled
#endif
```

Reply:
Yeah that's what I did on my inactive ASAN branch. https://github.com/slipher/Daemon/commit/237ff1b764d4eb56ef6ac64888d881992971575f